packages/terra-image/src/terra-dev-site/doc/example/ImageDefault.jsx

```diff
@@ -27,6 +33,16 @@ const ImageDefault = () => (
     <Image src="/mock/failure/route" alt="invalid src attribute image" placeholder={<Image src={placeholderPic1000x200} alt="placeholder image" />} />
     <h3>Valid src prop (src will show)</h3>
     <Image src={placeholderPic150x150} alt="src image" placeholder={<Image src={placeholderPic1000x200} alt="placeholder image" />} />
+    <br />
+    <h2>Fit</h2>
+    <h3>Cover</h3>
+    <div className={cx('image-container')}>
+      <Image className={cx('image')} src={coverImage170X251} alt="invalid src attribute image" Fit="cover" variant="circle" />
+    </div>
+    <h3>None</h3>
+    <div className={cx('image-container')}>
+      <Image className={cx('image')} src={coverImage170X251} alt="invalid src attribute image" Fit="none" />
+    </div>
```

Comment:
I'm not sure these examples are really illustrating the use of this prop, it looks more like a difference between circle and square.

Take a look at mdn for inspiration 
https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit

Reply:
fixed here https://github.com/cerner/terra-core/pull/2494/commits/be42fad09a1ef9b1e6caf2b5042500e0e494d4db.